mcp_server/tests/test_gds_functions.py

```diff
@@ -331,3 +331,54 @@ def test_projection_with_labels_and_types(neo4j_container):
 
     list_result = gds.graph.list()
     assert len(list_result) == 0
+
+
+@pytest.mark.asyncio
+def test_get_labels_and_types_and_properties(neo4j_container):
+    """Import test data into Neo4j."""
+    # Set environment variables for the import script
+    os.environ["NEO4J_URI"] = neo4j_container
+    os.environ["NEO4J_USERNAME"] = NEO4J_USER
+    os.environ["NEO4J_PASSWORD"] = NEO4J_PASSWORD
+
+    driver = GraphDatabase.driver(neo4j_container, auth=(NEO4J_USER, NEO4J_PASSWORD))
+    existing_count2 = -2
+    gds = GraphDataScience(driver)
+    with driver.session() as session:
+        session.run(
+            "CREATE (:Foo{prop1:1})-[:R1{relprop1:2}]->(:Bar), (:Bar)-[:R2{relprop2:2}]->(:Bar{prop2:2})"
```

Comment:
I think we want to extend this to cover the most general graph.
I believe the only missing case is multi-labelled nodes.